tests/testthat/test-predict-boots.R

```diff
@@ -122,32 +129,71 @@ test_that("predict_boots() throws an error when bad n is specified", {
 })
 
 test_that("predict_boots() throws an error when training_data/new_data doesn't match expected format", {
+  skip_if_not_installed("xgboost")
+  skip_if_not_installed("parsnip")
+  suppressPackageStartupMessages(library(workflows))
+  suppressPackageStartupMessages(library(parsnip))
+
+  test_wf_fit <- fit(test_wf, test_train)
+
 
   # predictors & outcome missing from training_data
-  expect_error(
+  expect_snapshot(
     predict_boots(
-      workflow = test_wf,
+      workflow = test_wf_fit,
       n = 1,
       training_data = test_train[, 3],
       new_data = test_test
     ),
-
-    paste0("missing cols in training_data:\n",
-           "species, island, bill_length_mm, bill_depth_mm, flipper_length_mm, sex, body_mass_g")
+    error = TRUE
   )
 
   # predictors missing from new_data
-  expect_error(
+  expect_snapshot(
```

Comment:
I moved most of the `expect_error()` code to `expect_snapshot()` since it tends to be more robust to unrelated changes to the package.